homeassistant/components/todo/services.yaml

```diff
@@ -42,5 +42,6 @@ remove_item:
         - todo.TodoListEntityFeature.DELETE_TODO_ITEM
   fields:
     item:
+      required: true
       selector:
-        object:
+        text:
```

Comment:
This parameter should allow multiple items.

Reply:
Correct, it does, however, we don't always provide multi-input selectors for these cases.
This element accepts both a single item and a list of items.

We have a task for the frontend to create/extend the text select to allow for multiple input elements in a next release.

Reply:
Ok.